README.md

```diff
@@ -818,6 +819,12 @@ to change Zappa's behavior. Use these at your own risk!
 ```javascript
  {
     "dev": {
+        "alb_enabled": false // enable provisioning of application load balancing resources
+        "alb_vpc_config": {
+            "CertificateArn": // ACM certificate ARN for ALB
```

Comment:
We try to keep the reference as legal JSON so it can be used as a template for a configuration.

```suggestion
            "CertificateArn": "arn_of_the_certificate", // ACM certificate ARN for ALB
```